src/components/Notification/Notification.test.jsx

```diff
@@ -32,6 +33,58 @@ const { Event, EventCloseAlert, EventButtons, EventCounter } =
 describe("Notification", () => {
   const user = userEvent.setup();
 
+  describe("Notification SSR behavior", () => {
+    const dateTimeOptions = {
+      year: "numeric",
+      month: "2-digit",
+      day: "2-digit",
+      hour: "2-digit",
+      minute: "2-digit",
+    };
+
+    it("reverts to default value for `locale` prop when navigator object is undefined", () => {
+      const { DateTimeFormat } = Intl;
+
+      vi.spyOn(Intl, "DateTimeFormat").mockImplementation(
+        (locale, options) => new DateTimeFormat(locale, options),
+      );
+
+      let renderResult = render(<Notification />);
+
+      const { container } = renderResult;
+
+      expect(container).toBeDefined();
```

Comment:
I'd argue that you don't need any of this. Just call `render` and be done. Your other tests take care of ensuring that the component loads properly. 
```suggestion
      render(<Notification />);
```

Reply:
If Yang's recommendations end up being the better way to test this, these comments will no longer apply.